### PR TITLE
Mig16.0 purchase requisition line description

### PR DESCRIPTION
**Pull request by laszlokr**
@janikvonrotz currently there is a bug that the description is copied twice in the purchase order lines. I tried changing and removing the `_prepare_purchase_order_line` method, but it's still copied twice

### CODE REVIEW

File: purchase_requisition_line_description/models/purchase_requisition.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+            # If no purchase description is given, set name
+            elif line.product_id:
+                line.product_description_variants += line.product_id.name
+
+            # Append supplier product name
+            if supplier_info.product_name:
+                line.product_description_variants += "\n" + supplier_info.product_name
+
+    # def _prepare_purchase_order_line(self, name, product_qty=0.0, price_unit=0.0, taxes_ids=False):
```
Comment (janikvonrotz) [inline purchase_requisition_line_description/models/purchase_requisition.py:57]: The commented code can be removed.

**Review by janikvonrotz — DISMISSED** [review]

**Comment by janikvonrotz** [comment]
@laszlokr What do you mean with "copied twice"? Is the computed method executed twice or is the description shown two times in field?

**Comment by laszlokr** [comment]
> @laszlokr What do you mean with "copied twice"? Is the computed method executed twice or is the description shown two times in field?

The description is shown two times in the field. I understand there is a system parameter to show and hide it, but I could not verify that it works, or I am testing it wrong

**Comment by janikvonrotz** [comment]
The system parameter only hides the product default code.

**Comment by janikvonrotz** [comment]
I sugget to add some loggers and try to figure out the behaviour of the computed method.

**Comment by laszlokr** [comment]
> I sugget to add some loggers and try to figure out the behaviour of the computed method.

```
2024-08-28 12:11:45,207 9506 WARNING 16.0 odoo.addons.purchase_requisition_line_description.models.purchase_requisition: Initial name in purchase order line: [FURN_8855] Schublade
DRAWER
DRAWER 
2024-08-28 12:11:45,207 9506 WARNING 16.0 odoo.addons.purchase_requisition_line_description.models.purchase_requisition: Product description variants already included in the name. 
```
It seems like the description is copied twice even before the `_prepare_purchase_order_line` method is executed

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**